lib/set.rb

```diff
@@ -38,27 +37,25 @@
 # * When a string is to be stored, a frozen copy of the string is
 #   stored instead unless the original string is already frozen.
 #
-# ## Comparison
+# == Comparison
 #
-# The comparison operators `<`, `>`, `<=`, and `>=` are implemented as
-# shorthand for the {proper_,}{subset?,superset?} methods.  The `<=>`
-# operator reflects this order, or return `nil` for sets that both
-# have distinct elements (`{x, y}` vs. `{x, z}` for example).
+# The comparison operators <tt><</tt>, <tt>></tt>, <tt><=</tt>, and <tt>>=</tt> are implemented as
```

Comment:
@BurdetteLamar I am curious, does the `<tt><</tt>` render into a legible `&lt;` (uh, I mean "does RDoc do the right thing", here?)